catalog/meta-chatbot-inaccuracies.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Meta's AI bot trained on scientific info put out biased and inaccurate articles
+companies: ["Meta"]
+categories: ["Inaccuracies", "Harmful Bias", "Misrepresentation"]
+demonstrative: false
+modalities: ["Chatbot"]
+date: 2022-11-22
+models: []
```

Comment:
There is a model here, no?

```suggestion
models: ["Galactica"]
```

Reply:
I initially was under the impression that Galactica is the name of the chatbot and the underlying model is something else (just like Bard is the name of Google's chatbot and the underlying model is LaMDA). But quick google search confirmed that Galactica is indeed the underlying model as well. Good catch.